draft-ietf-quic-transport.md

```diff
@@ -1143,40 +1125,30 @@ expectation that it will eventually receive an Initial packet.
 
 ## Handling Version Negotiation Packets {#handle-vn}
 
-When the client receives a Version Negotiation packet, it first checks that the
-Destination and Source Connection ID fields match the Source and Destination
-Connection ID fields in a packet that the client sent.  If this check fails, the
-packet MUST be discarded.
+How a client reacts to receiving a Version Negotiation packet is left as future
+work defined by future versions of QUIC.  Future versions of QUIC that define
```

Comment:
This isn't right.  This version needs to define what to do.

I recommend having clients abandon a connection attempt in the absence of better advice.